lib/hydra/pcdm/models/concerns/collection_behavior.rb

```diff
@@ -86,6 +86,13 @@ def ancestor? collection
       false
     end
 
+    def to_solr(solr_doc = Hash.new, opts={})
+      solr_doc = indexing_service.generate_solr_document
+      solr_doc["objects_ssim"] = self.objects.map { |o| o.id }
```

Comment:
Check out this commit: https://github.com/curationexperts/alexandria-v2/commit/91f3c61346f99b10b8b374599e1b34c56de2f81b Extend ActiveFedora::IndexingService and override generate_solr_document in it. Then override "Collection.indexer" to return the class you've defined.